shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/data/network/helpers/UrlPath.kt

```diff
@@ -0,0 +1,7 @@
+package co.nimblehq.blisskmmic.data.network.helpers
+
+import co.nimblehq.blisskmmic.BuildKonfig
+
+fun urlPath(path: String): String {
+    return "${BuildKonfig.BASE_URL}${path}"
```

Comment:
I think we can turn this into a localize resource string in `strings.xml` instead for better string resource management and reuse. Here is a good example of what I mean: 🙏
![Screen Shot 2022-09-27 at 15 19 13](https://user-images.githubusercontent.com/70877098/192472733-576a8091-8a70-46a9-a3c0-9f03c5222f9b.png)

What do you think? 😁

Reply:
I will remove this file as it's not used in the future PRs. 🙏

Reply:
oh do you mean you will remove this file in this PR right away or future PRs? 🤔